### PR TITLE
Fix styling for the dApp Staking top page hero area on mobile

### PR DESCRIPTION
**Pull request by sirius651**
**Pull Request Summary**

> Fix styling for the dApp Staking top page hero area on mobile

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

스크린샷 2022-12-01 오후 1 06 19

### CODE REVIEW

**Comment by sirius651** [comment]
@Kahonnohak I've fixed the css on mobile view. could you check if it's modified as expected?

**Comment by Kahonnohak** [comment]
Thanks!

I forgot to share the Figma link : https://www.figma.com/file/oXRfGoOgf6Yp84wJhwdvRg/dApp-Staking-V2-Kaizen?node-id=606%3A13980&t=MEp4o5ZmFrdN7D90-1

Let's make it clear one by one

1: The space here, is the same as before: if line brakes then the total height will change but the padding stay the same.
2: Can we remove this padding please and let people know that the contents extend?
3: The bottom padding is also same as before. My suggestion was to align at the bottom but we can have a look at it when the top align like you did now.
4: Could you give it a little more line space here please? It looks like there are two blocks so I cannot say the line height but maybe to have space of about 6px?

I hope it makes sense and ping me if anything is not clear.

**Comment by sirius651** [comment]
스크린샷 2022-12-01 오후 9 22 52

@Kahonnohak  fixed as you suggested. could you check again?

**Comment by Kahonnohak** [comment]
Hi thank you lets sync tomorrow when I wake up!

- I did not want to change the title location but wanted to use more space towards the bottom - I thought we might still have the image above but let me know if that is the issue
- Could you please remove the side space and show the contents till the edge (right side) 

スクリーンショット 2022-12-01 13 50 51

Does it make sense?

**Comment by sirius651** [comment]
> Does it make sense?

I fixed as you suggested. please check and if there's something let's sync

**Comment by sirius651** [comment]
@Kahonnohak I believe fixed it on feedback. please check

**Comment by Kahonnohak** [comment]
Lovely!! Thank you!

Last request. 
- Could you make the width even for each box? (at the moment each box has different width)
- It would be good if you could break the line for the Era as well? 

スクリーンショット 2022-12-02 13 55 50
スクリーンショット 2022-12-02 13 56 18

Getting there!

**Comment by sirius651** [comment]
> Getting there!

Fixed in this [PR](https://github.com/AstarNetwork/astar-apps/pull/612/commits/b22201d3e474d169e69ace6b494bc48aa4948764)

**Review by Kahonnohak — APPROVED** [review]
It looks awesome thank you so much!

**Comment by impelcrypto** [comment]
@sirius651 @Kahonnohak 
Perhaps add a padding or make font size slightly smaller here (only when the browser width is 375px)?

image

**Comment by impelcrypto** [comment]
maybe like this?
Screenshot 2022-12-06 at 1 39 41 PM

**Comment by Kahonnohak** [comment]
Great!

**Review by bobo-k2 — APPROVED** [review]

**Comment by sirius651** [comment]
> maybe like this?

Fixed as you suggested

**Review by impelcrypto — CHANGES_REQUESTED** [review]
Could you use @media (max-width:) instead of max-width in this file? Other than that, LGTM!

File: src/components/dapp-staking-v2/my-staking/styles/top-metric.scss [hunk 1]
```diff
@@ -126,23 +128,37 @@
...
+        min-width: 160px;
+      }
+    }
   }
 
   .column--era-info {
     display: flex;
     column-gap: 4px;
+
+    @media (max-width: $sm) {
```
[the remark this answers is not in the document]
Comment (sirius651) [reply src/components/dapp-staking-v2/my-staking/styles/top-metric.scss]: Fixed in [here](https://github.com/AstarNetwork/astar-apps/pull/612/commits/d7f9ef0ad6ad3a4ca97edf81f5ea2cdf3ee6b2e5)

**Comment by impelcrypto** [comment]
@sirius651 It looks like my CLI is having an issue to fix the conflicts. Could you help resolve the conflicts and merge this PR? I think you can delete the 2 conflict files:

 src/components/dapp-staking-v2/my-staking/TopMetric.vue
src/components/dapp-staking-v2/my-staking/styles/top-metric.scss

**Pull request merged**